include/monitor.h

```diff
@@ -3,7 +3,7 @@
 #include <wayland-server.h>
 #include <wlr/types/wlr_box.h>
 #include <wlr/types/wlr_output_damage.h>
-
+#include "server.h"
```

Comment:
We leave a space between the first block and the second one on purpos pls add one